models/marts/__models.yml

```diff
@@ -0,0 +1,21 @@
+
+version: 2
+
+models:
+  - name: customers
+    description: Customer overview data mart, offering key details for each unique customer. One row per customer.
+    columns:
+      - name: customer_id
+        description: The unique key of the orders mart.
+        tests:
+          - not_null
+          - unique
+
+  - name: orders
+    description: Order overview data mart, offering key details for each order inlcluding if it's a customer's first order and a food vs. drink item breakdown. One row per order.
+    columns:
+      - name: order_id
+        description: The unique key of the orders mart.
+        tests:
+          - not_null
+          - unique
```

Comment:
Are we leaving the description for the other columns null? I think the catalog partners would probably be able to show off more value if every column were defined - or maybe 50% because all of them can be a bear.

Reply:
ah good call, on the marts this makes sense -- on it! 🖖🏻